cli/0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4.json

```diff
@@ -0,0 +1,106 @@
+        "5737486047618054308558714882305655688513685593522322574530913512727366290642",
+        "1546514228629419350841439417596984741217702185191841354302016975094448618001",
+        "8316446845943768935143899259801711254165013914735479776722162070662579873358",
+        "12406235811057717974709223064796354493709890094496034214576083353124455871713",
+        "26932135483245270736354682342118303910943236465860172679954690638598898903281",
+        "29087056450339712589524148660228380701878500526032842382152257290568465182062",
+        "34052744657585017357155378025365939174585728722792788226033122047704822902233",
+        "8523674674116829203866405850572715682034988927429235234044210420284011007533"
+      ]
+    },
+    {
+      "encPubKey": "macipk.af1bd1abc6abae79463d706f0a3e4973901d5c0793806525074aac0092763392",
+      "iv": "8478144110375833309026707335061868192142899518121001972641903674528760003644",
+      "data": [
+        "7323464420517092632712600083321655338471998276714727457140333354189581751584",
+        "35025766326275108793144354364669673324822569804264533989808143312356463496060",
+        "18408416948647793649322638344759859822333548353342837916927832899671443766349",
+        "5648499278060284620677367856351849920077695502796431527140512790098198646750",
+        "14247319238842623361633966167244824981533076503653361450660217849439064260407",
+        "18554119485962898695202099213164460099624684623430334115219702606537966447970",
+        "7768620364149054119673282507211940564425758999888337165380526180563765775797",
+        "36142214601648639079849468953549289074006790350108808155997065530707824745812",
+        "23586199035158675002521611814616851808231770163460911758411519931865413030708",
+        "7933918181253224621322472012907377986092675361864562735170801184835115093784"
+      ]
+    },
+    {
+      "encPubKey": "macipk.d4769dc1c664d482938ea0046ae0c699e0a9f59395bb4093aea4c62a89b4041b",
+      "iv": "4513187001508452039022865128004328548053796599123590713647903069157971652180",
+      "data": [
+        "7009717873583378341796256238913354538845395049999329099405787492723430788924",
+        "24136764692481575610825198981740082039529252668171072985609809438521724945896",
+        "6021246186015476052392195551678248554579122174399044246198235862974731845119",
+        "8047622090333274689873485672915297739037639602108862360991057761549401979671",
+        "17963171418159573443212685830696624129145451976211369253153464087551051564356",
+        "18070900453130066079508991220033940484915745849682676584495920689808056185273",
+        "15281542088681398561172262336380123657983489309022367846766670534480332247250",
+        "15658976556044653017639073481670465795456937976326439654217215964137770110765",
+        "16804117294521997319830729223093417549221914353374389420572241640856758290235",
+        "12273892878991597886379704231168287877218070804087754512256378117154414171928"
+      ]
+    }
+  ]
+}
```

Comment:
It seems that this file was added inadvertently.

Reply:
Oops! I'll remove it.